src/components/Cancel.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import CancelLogo from "../assets/ic_X@3x.png";
+
+function Cancel({ value, onCancel }) {
```

Comment:
```suggestion
function Cancel({ value, onCancel }) {
```
value가 어떤 타입의 데이터가 들어오는지 이름만 봐서는 알기 어렵습니다.
매개변수를 구체적으로 적어주시고, default parameter 같은 기능으로 값이 없을때에 상황을 고려하는것도 좋은 방법입니다.